model_to_project_task/README.rst

```diff
@@ -0,0 +1,66 @@
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Model to Project Task
+=====================
+
+This module allows to end users to create an task from any configured models
+and link automatically the initial object to the task with a reference field.
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+* go to a model or object in tree or form view configured (ie Partner ou Product).
+* select a record (if your are in tree view).
+* select 'Define a task' in the 'More' button.
+* the task and its 'Task Origin' field is set
```

Comment:
the task is created and the field 'Task Origin' is linked to it in the original object